test/harnesses/EverlongTest.sol

```diff
@@ -37,51 +57,92 @@ contract EverlongTest is HyperdriveTest, IEverlongEvents {
+
+    // ╭───────────────────────────────────────────────────────────────────────╮
+    // │                           Everlong Storage                            │
+    // ╰───────────────────────────────────────────────────────────────────────╯
+
+    uint256 lastStrategyReportTime;
+    uint256 lastVaultReportTime;
+
+    /// @dev Time period for the strategy to release profits over.
+    uint256 internal STRATEGY_PROFIT_MAX_UNLOCK_TIME = 0;
+
+    /// @dev Time period for the strategy to release profits over.
+    uint256 internal VAULT_PROFIT_MAX_UNLOCK_TIME = 3 days;
+
+    /// @dev Everlong asset.
+    IERC20 internal asset;
 
     /// @dev Everlong token name.
     string internal EVERLONG_NAME = "Everlong Testing";
 
     /// @dev Everlong token symbol.
-    string internal EVERLONG_SYMBOL = "evTest";
+    string internal EVERLONG_SYMBOL = "EVRLNG";
 
-    uint256 internal TARGET_IDLE_LIQUIDITY_PERCENTAGE = 0.1e18;
-    uint256 internal MAX_IDLE_LIQUIDITY_PERCENTAGE = 0.2e18;
+    // TODO: Implement and test.
+    uint256 internal TARGET_IDLE_LIQUIDITY_BASIS_POINTS = 0;
+    uint256 internal MIN_IDLE_LIQUIDITY_BASIS_POINTS = 0;
 
-    IEverlong.RebalanceOptions internal DEFAULT_REBALANCE_OPTIONS =
-        IEverlong.RebalanceOptions({
-            spendingLimit: 0,
-            minOutput: 0,
-            minVaultSharePrice: 0,
-            positionClosureLimit: 0,
-            extraData: ""
-        });
+    /// @dev Everlong vault management address.
+    /// @dev Used when interacting with the `DebtAllocator`.
+    address internal management;
 
-    // ╭─────────────────────────────────────────────────────────╮
-    // │ Hyperdrive Configuration                                │
-    // ╰─────────────────────────────────────────────────────────╯
+    /// @dev Everlong keeper address.
+    address internal keeper;
 
-    address internal HYPERDRIVE_INITIALIZER = address(0);
+    /// @dev Mainnet `RoleManager` factory.
+    IRoleManagerFactory internal roleManagerFactory =
+        IRoleManagerFactory(0xca12459a931643BF28388c67639b3F352fe9e5Ce);
 
-    uint256 internal FIXED_RATE = 0.05e18;
-    int256 internal VARIABLE_RATE = 0.10e18;
+    /// @dev Everlong strategy factory.
+    IEverlongStrategyFactory internal strategyFactory;
 
-    uint256 internal INITIAL_VAULT_SHARE_PRICE = 1e18;
-    uint256 internal INITIAL_CONTRIBUTION = 2_000_000e18;
+    /// @dev Everlong strategy.
+    IEverlongStrategy internal strategy;
 
-    uint256 internal CURVE_FEE = 0.01e18;
-    uint256 internal FLAT_FEE = 0.0005e18;
-    uint256 internal GOVERNANCE_LP_FEE = 0.15e18;
-    uint256 internal GOVERNANCE_ZOMBIE_FEE = 0.03e18;
+    /// @dev Everlong vault.
+    IVault internal vault;
+
+    IRoleManager internal roleManager;
+    DebtAllocator internal debtAllocator;
+    Registry internal yearnRegistry;
+    IAccountant internal accountant;
+    IAprOracle internal aprOracle =
+        IAprOracle(0x1981AD9F44F2EA9aDd2dC4AD7D075c102C70aF92);
```

Comment:
Same here. Some documentation would make this easier to parse for those not as familiar with yearn (myself included).